nodes/src/nodes/dataset_cobalt/IGlobal.py

```
@@ -0,0 +1,166 @@
+                    warning(f'Cobalt Dataset Global: {str(e)}')
+                    return
+
+                if not os.path.isfile(normalized):
+                    warning(f'Cobalt Dataset Global: File not found: {normalized}')
+                    return
+
+            if sample_size < 0:
+                warning(f'Cobalt Dataset Global: sample_size must be >= 0, got {sample_size}')
+                return
+
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)